test/test-utils.js

```diff
@@ -0,0 +1,35 @@
+import chai from 'chai';
+import nock from 'nock';
+import { baseUrl } from '../src/utils';
+
+///////////////////////////////////////////////////////////////////////////////
+// Short-hand for comparing two results when testing:
+
+const equal = (a, b) => {
+  return chai.expect(a).to.deep.equal(b);
+};
+
+///////////////////////////////////////////////////////////////////////////////
+// Wrapper for nock which ignores the final part of the URI. If not using this
+// wrapper unexpected results can occur, the most evident being that the
+// following two URLS are considered different:
+//
+// http://www.example.com/?arg1=foo&arg2=bar
+// http://www.example.com/?arg2=bar&arg1=foo
+//
+// ..and also the following two URLs:
+//
+// http://www.example.com/?arg1=foo&arg2=bar
+// http://www.example.com/%3Farg1%3Dfoo%26arg2%3Dbar
```

Comment:
Nock's probleem is dat de semantiek van een URL er niet toe doet bij het mocken van API calls, alleen de exacte string. In de deze comment staan maar twee voorbeeldjes van waar Nock afwijkt van het verwachte gedrag: we hebben zelf samen nog een derde voorbeeld gezien waarbij twee schijnbaar identieke URLs door Nock als verschillend gezien werden.